src/backends/gaston.jl

```diff
@@ -0,0 +1,120 @@
+# https://github.com/mbaz/Gaston.
+const G = Gaston
+const GastonSubplot = G.Plot
+# --------------------------------------------
+# These functions are called by Plots
+# --------------------------------------------
+# Create the window/figure for this backend.
+function _create_backend_figure(plt::Plot{GastonBackend})
+    state_handle = G.nexthandle() # for now all the figures will be kept
+    plt.o = G.newfigure(state_handle)
+end
+
+
+function _before_layout_calcs(plt::Plot{GastonBackend})
+    # Initialize all the subplots first
+    plt.o.subplots = G.SubPlot[]
+
+    grid = size(plt.layout)
+    plt.o.layout = grid
+
+    for sp in plt.subplots
+        gaston_init_subplot(plt, sp)
+    end
+
+    # Then add the series (curves in gaston)
+    for series in plt.series_list
+        gaston_add_series(plt, series)
+    end
+end
+
+function _update_min_padding!(sp::Subplot{GastonBackend})
+    sp.minpad = (20mm, 5mm, 2mm, 10mm)
+end
+
+function _update_plot_object(plt::Plot{GastonBackend})
+end
+
+function _show(io::IO, ::MIME"image/png", plt::Plot{GastonBackend})
```

Comment:
saving figures would be done through this function

Reply:
@mbaz it would be really helpful if you could help me fill this cuntion correctly